databases/core.py

```diff
@@ -15,8 +15,10 @@
 
 if sys.version_info >= (3, 7):  # pragma: no cover
     from contextvars import ContextVar
+    import contextvars as contextvars
```

Comment:
Can L17, L18 imports be merged?

Reply:
Is there a format to do this in a single line? :-o
Probably "explicit is better than implicit" principle should be applied here and ContextVar should be contextvars.ContextVar to emphasize the dependency.  Of course we could also import `copy_context` since this is the only function we need, but (i think) using foreign namespace directly is not very fortunate.

Reply:
@goteguru 
Sorry if my comment was confusing :D
What I mean is that can we do `import contextvars` and get `contextvars.ContextVars` and `contextvars.copy_context`?

Reply:
@goteguru what do you think about my comment?
If you don't have the time I can do that change, but your input would be appreciated. I look forward to get this through.